Backend/utils/db_utils.py

```diff
@@ -0,0 +1,224 @@
+import json
+import boto3
+import MentiiLogging as MentiiLogging
+from boto3.dynamodb.conditions import Key, Attr
+from botocore.exceptions import ClientError
+
+logger = MentiiLogging.getLogger()
+
+def createTableFromFile(settings_file, dbInstance):
+  try:
+    with open(settings_file) as f:
+      data = f.read()
+      settings = json.loads(data)
+
+      attribute_definitions = settings.get("AttributeDefinitions")
+      key_schema = settings.get("KeySchema")
+      provisioned_throughput = settings.get("ProvisionedThroughput")
+      table_name = settings.get("TableName")
+
+      # check for required parameters
+      if table_name is not None and key_schema is not None and attribute_definitions is not None and provisioned_throughput is not None:
+        if len(key_schema) > 0 and len(attribute_definitions) > 0:
+          table = dbInstance.create_table(
+              AttributeDefinitions=attribute_definitions,
+              KeySchema=key_schema,
+              ProvisionedThroughput=provisioned_throughput,
+              TableName=table_name
+            )
+          f.close()
+          return table
+      f.close()
+      raise(IOError)
+  except IOError as e:
```

Comment:
We don't need to catch an IOError if we're rasing it. Using with removes the need to catch file opening exceptions. Try catch statements are really expensive in python and this is an easy thing to fix.